coops-ui/src/interfaces/BCorpsIF.ts

```diff
@@ -0,0 +1,4 @@
+export interface ARDateIF {
```

Comment:
Is this used anywhere?

Reply:
Thanks Severin, good catch. In this current Iteration the interface was no longer required after the way the dates were output changed. I will remove this interface, rename the file to `interfaces.ts` for us to use in the ( near ) future.